index.html

```diff
@@ -5850,11 +5864,16 @@ <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
                   false).
                 </li>
                 <li>Perform <a href=
-                "https://tc39.github.io/ecma262/#sec-createdataproperty">CreateDataProperty</a>(O=<var>this</var>,
-                P=<var>paramName</var>, V=<var>audioParam</var>).
+                "https://tc39.github.io/ecma262/#https://tc39.github.io/ecma262/#sec-set-o-p-v-throw">
+                  Set</a>(O=<var>parameterMap</var>, P=<var>paramName</var>,
+                  V=<var>audioParam</var>).
                 </li>
               </ol>
             </li>
+            <li>Perform <a href=
+            "https://tc39.github.io/ecma262/#sec-createmethodproperty">CreateDataProperty</a>(O=<var>this</var>,
+            P="parameters", V=<var>parameterMap</var>).
+            </li>
```

Comment:
See here: https://rawgit.com/hoch/web-audio-api/a4bf075d8e5ee79e28ba75ad9dce8c0eeb42dc74/index.html#AudioWorkletNode

The new IDL defines `AudioParamMap` interface which is a `readonly` maplike storage. The AudioWorkletNode also has this object as a `readonly` member.

Do we need anything else to protect it?